console/console-init/ui/src/modules/iot-device/dialogs/CreateDevice/DeviceInformation.tsx

```diff
@@ -35,12 +38,19 @@ export const DeviceInformation: React.FunctionComponent<IDeviceInfo> = ({
   const handleChange = () => {
     setIsChecked(!isChecked);
   };
+
+  const styles = StyleSheet.create({
+    lighter_text: {
+      fontWeight: "lighter"
```

Comment:
please use patterfly global variable for fontwieght. 
You can find all the PF variables here: https://patternfly-react.surge.sh/documentation/overview/global-css-variables
Try this one 
fontWeight: var("--pf-global--FontWeight--light")